media-libs/libggi/files/libggi-2.2.2-incompatible-types.patch

```diff
@@ -0,0 +1,11 @@
+--- a/display/X/helper/dga/dga.c	2024-03-27 20:58:09.954061371 +0400
```

Comment:
If you can, for these, please include a link at the top to the Gentoo bug.

This is important for a few reasons:
1) For verifying the patch is correct down the line (in a few years if someone asks why we have it);
2) Other distros are likely to pick up these patches and having some provenance is good